seeds/events.js

```diff
@@ -0,0 +1,59 @@
+const BPromise = require('bluebird');
+const util = require('../src/util/seeds');
+const fs = require('fs');
+const path = require('path');
+const logger = require('../src/util/logger');
+const DATA_DIR = path.join(__dirname, '../data');
+
+exports.seed = function(knex, Promise) {
+  return knex('cities')
+    .select('id')
+    .select('name')
+    .then(cities => cities.filter(city => _fileExists(city.name)))
+    .then(cities => {
+      return BPromise.map(cities, city => {
+        return BPromise.map(_readEvents(city), (event, index) => {
+          return util.insertOrUpdate(knex, 'events', {
+            id: `${ city.name }_${ index }`,
```

Comment:
This should be a `code` field. Update should be done using it in `WHERE` condition